packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -158,20 +158,67 @@ export default class WebpackPlugin extends PluginBase<WebpackPluginConfig> {
 
           this.isProd = true;
           await fs.remove(this.baseDir);
-          await listrCompatibleRebuildHook(
-            this.projectDir,
-            await getElectronVersion(this.projectDir, await fs.readJson(path.join(this.projectDir, 'package.json'))),
-            platform,
-            arch,
-            config.rebuildConfig,
-            task,
-            chalk.cyan('[plugin-webpack] ')
-          );
-        }, 'Preparing native dependencies'),
-        namedHookWithTaskFn<'prePackage'>(async () => {
-          await this.compileMain();
-          await this.compileRenderers();
-        }, 'Building webpack bundles'),
+
+          // TODO: Figure out how to get matrix from packager
+          let arches: string[] = [arch];
+          if (arch === 'universal') {
+            arches = ['arm64', 'x64'];
```

Comment:
Tbh hardcoding this is fine, making it dynamic is probably not worth the squeeze

Reply:
Yeah this is defs the 99% case, the 1% case is folks running `package --arch=all` but idk if that even works at the moment let alone with webpack 😅